src/core/namespace.c

```diff
@@ -4016,6 +4016,13 @@ DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_map_type, uint64_t);
 DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_prog_type, uint64_t);
 DEFINE_STRING_TABLE_LOOKUP(bpf_delegate_attach_type, uint64_t);
 
+static const char* const memory_thp_table[_MEMORY_THP_MAX] = {
+        [MEMORY_THP_DISABLE] = "disable",
+        [MEMORY_THP_MADVISE] = "madvise",
```

Comment:
very confused now. the enum knows 3 values, this one here only 2? what's going on?